zip-0316.rst

```diff
@@ -305,10 +325,10 @@ The raw encoding of a Unified Address is a concatenation of
 :math:`(\mathtt{typecode}, \mathtt{length}, \mathtt{addr})` encodings
 of the consituent Receivers:
 
-* :math:`\mathtt{typecode} : \mathtt{byte}` — the Typecode from the
+* :math:`\mathtt{typecode} : \mathtt{compactSize}` — the Typecode from the
   above Priority List;
 
-* :math:`\mathtt{length} : \mathtt{byte}` — the length in bytes of
+* :math:`\mathtt{length} : \mathtt{compactSize}` — the length in bytes of
```

Comment:
TODO(daira): this needs to be updated to be below the lower limit of zcashd's, that limits the length of compact size encoding to the number of elements (length encoding vs number encoding)